Morcoso_Daisy/Countries of the world/script.js

```diff
@@ -0,0 +1,78 @@
+const searchButton = document.getElementById("searchButton");
+const countryInput = document.getElementById("countryInput");
```

Comment:
Your ID's in the codes must be written in snake case. It should be all lowercase letters that are separated by an underscore.

For reference follow the guidelines:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5